README.md

```diff
@@ -73,7 +68,6 @@ Clone this repository to your desired folder:
   git clone https://github.com/aima98/OOP-School-Library.git
 ```
 
-```
 ### Run tests
 To run tests, run the following command:
```

Comment:
- [ ] As mentioned by the previous reviewer it could be better if you replace the Run tests section with the Run app section and add the command to run your application in the terminal `ruby main.rb` 😁 😁 
- [ ] Also the link to `Install` still not getting to the relevant section because that section doesn’t exist, if you can add it or delete the link to the table of contents 🙃 😁